docs/reference/doofinder.facetquery.size.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [doofinder](./doofinder.md) &gt; [FacetQuery](./doofinder.facetquery.md) &gt; [size](./doofinder.facetquery.size.md)
+
+## FacetQuery.size property
+
+The number of results to return in each field. Only applicable to facet terms. Maximum: 50 by default to help maintain the speed of the search function

```

Comment:
In my opinion, "to help maintaining" would be more correct

Reply:
done